pkg/daemon/ceph/config/config.go

```diff
@@ -48,6 +44,16 @@ const (
 	msgr2Prefix = "v2:"
 )
 
+var (
+	// DefaultConfigDir is the default dir where Ceph stores its configs. Can be overridden for unit
+	// tests.
+	DefaultConfigDir = "/etc/ceph"
```

Comment:
We have https://github.com/rook/rook/blob/master/pkg/operator/ceph/config/config.go#L70 can we use it instead?

Reply:
Oh I see.